src/uct/base/uct_cm.c

```diff
@@ -282,29 +282,31 @@ static ucs_stats_class_t uct_cm_stats_class = {
 #endif
 
 UCS_CLASS_INIT_FUNC(uct_cm_t, uct_cm_ops_t* ops, uct_iface_ops_t* iface_ops,
+                    uct_iface_internal_ops_t* internal_ops,
                     uct_worker_h worker, uct_component_h component,
                     const uct_cm_config_t* config)
 {
-    self->ops                     = ops;
-    self->component               = component;
-    self->iface.super.ops         = *iface_ops;
-    self->iface.worker            = ucs_derived_of(worker, uct_priv_worker_t);
-
-    self->iface.md                = NULL;
-    self->iface.am->arg           = NULL;
-    self->iface.am->flags         = 0;
-    self->iface.am->cb            = (uct_am_callback_t)ucs_empty_function_return_unsupported;
-    self->iface.am_tracer         = NULL;
-    self->iface.am_tracer_arg     = NULL;
-    self->iface.err_handler       = NULL;
-    self->iface.err_handler_arg   = NULL;
-    self->iface.err_handler_flags = 0;
-    self->iface.prog.id           = UCS_CALLBACKQ_ID_NULL;
-    self->iface.prog.refcount     = 0;
-    self->iface.progress_flags    = 0;
-
-    self->config.failure_level    = config->failure;
-    self->config.reuse_addr       = config->reuse_addr;
+    self->ops                      = ops;
+    self->component                = component;
+    self->iface.super.ops          = *iface_ops;
+    self->iface.internal_ops       = internal_ops;
+    self->iface.worker             = ucs_derived_of(worker, uct_priv_worker_t);
+
+    self->iface.md                 = NULL;
+    self->iface.am->arg            = NULL;
+    self->iface.am->flags          = 0;
+    self->iface.am->cb             = (uct_am_callback_t)ucs_empty_function_return_unsupported;
+    self->iface.am_tracer          = NULL;
+    self->iface.am_tracer_arg      = NULL;
+    self->iface.err_handler        = NULL;
```

Comment:
this whole block can be moved one space left, as it was before

Reply:
@brminich Good catch. Thanks!